slack/resource_conversation.go

```diff
@@ -141,6 +159,54 @@ func resourceSlackConversationCreate(ctx context.Context, d *schema.ResourceData
 	return resourceSlackConversationRead(ctx, d, m)
 }
 
+func findExistingChannel(ctx context.Context, client *slack.Client, name string, isPrivate bool) (*slack.Channel, error) {
+	// find the existing channel. Sadly, there is no non-admin API to search by name,
+	// so we must search through ALL the channels
```

Comment:
An extreme sadness with the existing splunk api.  There is an admin api for searching for a specific channel, but that would _greatly_ limit the deployment ability of this provider, as it would need to be configured with an administrator user token.